### ISSUE TITLE
How do I stop an event?

### ISSUE THREAD

**Issue by ColdlordXRepositories** [ShadowMario/FNF-PsychEngine#11074]
### Describe your problem here.

I'm trying to make an event to change the window size (height and width) but something is going wrong.
I have two other events, a window X adjuster, and a window Y adjuster. These are good (they work perfect) except for one thing... When I run an event, they all run at the same time.
Here is my code:
(Window X)
function onCreate()
    setPropertyFromClass("openfl.Lib", "application.window.x", "300" )
    function onEvent(name, value1, value2)
      setPropertyFromClass("openfl.Lib", "application.window.x", (value1))
    end
end
(Window Y)
function onCreate()
    setPropertyFromClass("openfl.Lib", "application.window.y", "150" )
    function onEvent(name, value1, value2)
      setPropertyFromClass("openfl.Lib", "application.window.y", (value1))
    end
end
(Window size)
function onCreate()
    setPropertyFromClass("openfl.Lib", "application.window.width", "1280" )
    setPropertyFromClass("openfl.Lib", "application.window.height", "720" )
    function onEvent(name, value1, value2)
      setPropertyFromClass("openfl.Lib", "application.window.width", (value1) + "560")
      setPropertyFromClass("openfl.Lib", "application.window.height", (value1))
    end
end
Basically, when I run a single event, ALL of them play at once.
### Are you modding a build from source or with Lua?
Lua

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

bunch of events

**Comment by frantastic24** [comment]
`onCreate()` runs when loading the song, you want to use `onEvent()`
It's explained on the wiki

**Comment by frantastic24** [comment]
Why is it not in the code you posted, that's the part that actually matters
You probably didn't add event name checks

**Comment by ColdlordXRepositories** [comment]
> Why is it not in the code you posted, that's the part that actually matters You probably didn't add event name checks

It IS in the code. Are you not seeing it? 
![Screenshot 2022-10-31 200028](https://user-images.githubusercontent.com/49921877/199131033-8ebbba6c-e67f-4aa9-8b75-99ff575d37ce.png)

**Comment by TheKitBoi** [comment]
if name == 'thename' then
-- code
end

**Comment by ColdlordXRepositories** [comment]
> if name == 'thename' then -- code end

oh

> Why is it not in the code you posted, that's the part that actually matters You probably didn't add event name checks

sorry

**Comment by frantastic24** [comment]
Oops that's my bad, it's impossible to see when it's not properly formatted

**Comment by ColdlordXRepositories** [comment]
> Oops that's my bad, it's impossible to see when it's not properly formatted and the functions are inside each other (why??)

Yeah I did that because for some reason when I properly format it it gets rid of all the line breaks
At least for me

**Comment by ColdlordXRepositories** [comment]
> That's because you're not properly formatting it, that only works for single short lines, you want to put it in a box

Thank you!!
